### PR TITLE
Use CMAKE_INSTALL_FULL_*DIR locations for installation

### PR DESCRIPTION
**Pull request by tari01**

### CODE REVIEW

File: data/CMakeLists.txt [hunk 1]
```diff
@@ -75,19 +75,19 @@ endif()
...
-set(DBUS_IFACE_DIR "${CMAKE_INSTALL_PREFIX}/share/dbus-1/interfaces")
-set(ACCOUNTS_IFACE_DIR "${CMAKE_INSTALL_PREFIX}/share/accountsservice/interfaces")
+set(POLKIT_LIB_DIR "${CMAKE_INSTALL_FULL_LOCALSTATEDIR}/lib/polkit-1")
+set(POLKIT_DATA_DIR "${CMAKE_INSTALL_FULL_DATADIR}/polkit-1")
+set(DBUS_IFACE_DIR "${CMAKE_INSTALL_FULL_DATADIR}/dbus-1/interfaces")
+set(ACCOUNTS_IFACE_DIR "${CMAKE_INSTALL_FULL_DATADIR}/accountsservice/interfaces")
 
 install(FILES org.ayatana.indicator.sound.AccountsService.xml
-	DESTINATION "${DBUS_IFACE_DIR}"
+    DESTINATION "${DBUS_IFACE_DIR}"
```
Comment (sunweaver) [inline data/CMakeLists.txt:84]: Please separate white-space changes and content changes and split them up into two commits. Thanks.

**Review by sunweaver — CHANGES_REQUESTED** [review]
White space vs. non-white-space changes. Split them up into two commits.

**Review by sunweaver — APPROVED** [review]
ACK.

**Comment by sunweaver** [comment]
I cannot simply rebase this branch due to conflicts with master, it seems. Please refurbish this PR: take latest master, cherry-pick changes of this PR on top, force-push to this PR's update branch.

**Comment by tari01** [comment]
> I cannot simply rebase this branch due to conflicts with master, it seems. Please refurbish this PR: take latest master, cherry-pick changes of this PR on top, force-push to this PR's update branch.

It should work - the changes it shows are the ones we want.

**Review by sunweaver — APPROVED** [review]
LGTM

**Pull request merged**